jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
```

Comment:
Ahh, I was looking for the `mktemp -t` behavior but I didn't realize this is what the docs meant by "template".  I would prefer this over `mkdir /tmp/...`, although why stop there? Let's keep going and get rid of the shell-out and just `Dir.mktempdir` like @rick kindly reminded us :D